### DISCUSSION TITLE
When is release of Elsa 2.0 planned?

### DISCUSSION THREAD

**Discussion by tomy2105** [elsa-workflows/elsa-core#766] [Q&A]
Not a bug but a "simple" question :). 
I saw mention of Q1 somewhere, but this is obviously not happening since Q1 is in 15 days :(.
So what is the new estimate, and I'm content with the best guess one.

**Comment by craigfowler** [comment]
As things stand the estimate has not been updated since you last heard.  The general feeling I have at the moment is that _"It's still possible that we could be ready for that deadline"_.  Certainly my feeling is that _if the current estimated ready-date is missed_, it won't be by much.  A number of the issues open against 2.0 are either in a state where the work is done, they just need testing, or actually the issue might be deferred into a later release.  The remaining "major" bits of work relate only to the dashboard/designer UI, and progress is being made on those at pace.

The official line on that estimate is stored against [**the Elsa 2.0 milestone**](https://github.com/elsa-workflows/elsa-core/milestone/8).  That's also the first place where it will be updated if the situation changes.

**Reply by craigfowler** [reply]
# Milestone & estimated date updated

@tomy2105 - FYI this was just posted in the Discord announcements channel by @sfmskywalker 

> **Milestone Elsa 2.0 moved to April 30**
> 
> The Elsa 2.0 milestone was just updated from March 31 to April 30 to better reflect our current status, since it's unlikely we'd be able to release a proper 2.0 version end of this month.
Although we've made great headway, it has become clear that we simply need more time to build a proper designer.
> 
> My expectation however is still that we can at least ship a preview version of the dashboard + designer that will already be more useful than that of Elsa 1. But there are a number of quality issues that need to be resolved before we're ready to assign it the 2.0 stable tag.

**Reply by tunechr** [reply]
Great thanks for the update craig.

**Reply by tomy2105** [reply]
Thanks for the update.
